src/d3a/d3a_core/sim_results/area_statistics.py

```diff
@@ -592,3 +591,41 @@ def _convert_output_format(self, price_energy, csv_output, redis_output):
                 } for timeslot, trades in trade_rates.items()
             ]
             redis_output[node.uuid] = deepcopy(csv_output[node.name])
+
+
+class BaselinePeakEnergyStats:
+    def __init__(self):
+        self.baseline_peak_percentage_result = {}
+        self.baseline_peak_percentage_result_redis = {}
```

Comment:
Are there any expected difference of results to be sent to UI and backend reports ?

Reply:
Yes, just like in all other stats: area names are replaced by area uuids and numbers are rounded

Reply:
Thanks for reminding, in that case it looks good